atlasdb-client/src/main/java/com/palantir/atlasdb/AtlasDbConstants.java

```diff
@@ -84,7 +84,7 @@ private AtlasDbConstants() {
     public static final long DEFAULT_TRANSACTION_LOCK_ACQUIRE_TIMEOUT_MS = 60_000;
     public static final int THRESHOLD_FOR_LOGGING_LARGE_NUMBER_OF_TRANSACTION_LOOKUPS = 10_000_000;
 
-    public static final Set<TableReference> hiddenTables = ImmutableSet.of(
+    public static final Set<TableReference> HIDDEN_TABLES = ImmutableSet.of(
```

Comment:
We should follow-up with tracking down all tables that should be in the respective sets. Ideally, we should add this to table metadata because the current model is definitely not working